src/v/config/configuration.cc

```diff
@@ -689,6 +689,18 @@ configuration::configuration()
       "Enable archival storage",
       {.visibility = visibility::user},
       false)
+  , cloud_storage_enable_remote_read(
+      *this,
+      "cloud_storage_enable_remote_read",
+      "Enable remote read for all topics",
+      {.visibility = visibility::user},
```

Comment:
i think this can be marked as the property that do not require restart

Reply:
yep